ircdog.go

```diff
@@ -64,6 +64,7 @@ Sending Escapes:
 Options:
 	--tls               Connect using TLS.
 	--tls-noverify      Don't verify the provided TLS certificates.
+	--tls-cert=<file>   A file containing a TLS client cert & key, to use for TLS connections.
```

Comment:
Sorry, one more thing, could you name this --client-cert? We are also considering a server cert argument in #9.

Reply:
`--client-cert` is long enough that it throws out the alignment of flag name vs description, btw